### ISSUE TITLE
.show-for-print not working as documented

### ISSUE THREAD

**Issue by cconover** [zurb/foundation#6445]
The CSS class `.show-for-print` is not being applied [according to the documentation](http://foundation.zurb.com/docs/components/typography.html#print-styles) when added to an element.

**What I'm doing**
I'm adding `show-for-print` to the `class` declaration of an element. In this example, I'll use an `<a>` tag.

    <a class="show-for-print" href="http://example.com"><img src="example.png" alt="Example image"></a> 

**What is happening**
The tag and its contents appear fine when printing, which is the expected behavior. However, according to the documentation the tag should be hidden when not printing. That is not happening. Inspecting the element reveals that no media query for print, nor any declaration of `.show-for-print`, is being made when viewing on screen.

**Comment by caycek** [comment]
So reading the documentation I 100% agree with you on the wording... If you want to manually fix this either manually add a 

@media not print{
    .show-for-print{ display: none; }
}

Or edit your foundation.css on line 6090 and add just

.show-for-print{ display: none; }

 This should resolve the issue.

For admins if this is unacceptable and not intended behavior please review the docs wording.

**Comment by cconover** [comment]
That's what I ended up doing. I added the following line to my `_settings.scss` file in the Media Queries section:

    $print: "only print";

Then I added the following lines to one of my main SCSS partials:

    @media #{$screen} {
        .show-for-print {
            display: none !important;
        }
    }
    
    @media #{$print} {
        .hide-for-print {
            display: none !important;
        }
    }

Your example drew my attention to the fact that my solution for anything not print isn't actually the right way, since it only accounts for screens. I think I will switch to your `not print` method instead.

The documentation and implementation definitely needs to be reviewed by admins.
